proc/proc_windows.go

```diff
@@ -127,6 +127,20 @@ func Launch(cmd []string) (*Process, error) {
 		return nil, ProcessExitedError{Pid: dbp.Pid, Status: exitCode}
 	}
 
+	for _, thread := range dbp.Threads {
+		_, err := _SuspendThread(thread.os.hThread)
```

Comment:
@lukehoban code didn't, [their documentation](https://msdn.microsoft.com/en-us/en-en/library/windows/desktop/ms686345.aspx) doesn't mention it, and it doesn't seem to cause problems.